innersource-activity-score.md

```diff
@@ -0,0 +1,92 @@
+## Title 
+InnerSource Activity Score
+
+## Patlet
+The InnerSource activity score is a numeric value that represents the (GitHub) activity of an InnerSource project. It can be used to create a ranked list of projects for the [InnerSource portal](innersource-portal.md) so that potential contributors can easily find the most active projects. It is derived automatically from GitHub repository statistics and may be enriched with KPIs from other tools or manual evaluations. 
+
+## Problem
+
+Potential contributors want to find active projects with a lot of traction, but also fairly new and enthusiastic projects with a need for new contributions - easy and fast.
+
+**But in which order** shall InnerSource projects be presented? Typical ranking KPIs like *GitHub Stars*, *Number of Forks*, *Number of Commits*, *Lines of Code*,  *Last Update* aren't very accurate when it comes to activity of a project.
```

Comment:
instead of "aren't very accurate", perhaps it is more appropriate to say "aren't sufficient to concisely indicate the activity of a project"